### ISSUE TITLE
Prism Subsite

### ISSUE THREAD

**Issue by brianmrock** [stoplightio/prism#426]
**User Story**

As a potential Prism user, I want to see a simple marketing-product style page explaining what prism is, so I know if it can help me or not.  

As  I would like this page to be part of the main website, so we get SEO benefits. https://stoplight.io/prism

We should make it perfectly clear prism is its own entity, do not talk about hosted prism at all on this page, this i the Open Source Prism for mocking on CLI and with JS API etc. Get people sharing this around because nothing else does this. 

**Acceptance Criteria**
- New subsite, standard product-style landing page
- Explain that its a CLI-based HTTP Server, and JS HTTP Client
- Screenshots of Prism CLI mocking
- Nice Examples of the JS API doing various things
- Needs to be in the stoplight.io react app

**Out of Scope**
- Writing Documentation, that will be done by others
- “Hubs” integration (just link to the GitHub Markdown)

**Comment by brianmrock** [comment]
Marc MacLeod
June 20, 2019, 9:31 PM

Phil Sturgeon couple of thoughts:

The term “microsite” might confuse folks (it confuses me at least). What we’re talking about here is just a subpage on the stoplight.io site right? Not a standalone microsite.

I don’t think we have to mention hosted mocking at all right now since it’s not really ready yet.

The stoplight.io source code is pretty simple react stuff over here: stoplightio/stoplight.io. Anybody can take a crack at putting something together that fits reasonably well UI wise with the rest of the stoplight site.

Thoughts?


Phil Sturgeon
June 21, 2019, 12:09 AM

Yep I recommended the URL  https://stoplight.io/open-source/prism 

And I’ve suggested we keep the concept of CLI/JS prism and hosted mocking very separate so absolutely, agreed with you there.


Phil Sturgeon
June 21, 2019, 1:14 PM

Changed the URL to https://stoplight.io/prism to line up with the Prism CLI errors.


Marc MacLeod
June 21, 2019, 2:30 PM

Hmm, I think having an open-source section of the main site would be nice. As we release more open source stuff, /open-source could serve as our landing page for all the open source projects. 

In any case, CLI errors should probably be updated to point to docs right? Can’t imagine we’d put description of errors on the marketing website.

On a separate note, one thing we CAN do on studio release is update prism subpage to mention awesome integration inside of studio, which is free.


Phil Sturgeon
June 21, 2019, 2:41 PM

It can go either way, we can change links because it’s only beta, or redirect, or do anything we like. 

I was thinking /studio, /spectral, /prism, /open-source, etc could all be top level because when we start mixing in bits like hosted spectral that’s less open source. We can still have an open source section which points off to our open source projects, but we might be better off having less specific URLs for our product landing pages 


Vincenzo Chianese
June 24, 2019, 10:07 AM

I just hopped on this conversation while taking ownership of the task, but I’m not too sure I understand the final decision.

Are we going to have a separate website (still under Stoplight.io domain) for Prism? Or is it going to be just a subpage?

 


Marc MacLeod
June 24, 2019, 1:29 PM
Edited

Subpage (site is wrong word) on stoplight.io. It’s a simple react static site over here: stoplightio/stoplight.io 


Phil Sturgeon
June 24, 2019, 5:15 PM

> I would like this page to be part of the main website, so we get SEO benefits.  https://stoplight.io/prism 

It says this right at the top of the story 😅
